adagrams/game.py

```diff
@@ -1,13 +1,126 @@
 from random import randint
 
 def draw_letters():
-    pass
+    tiles = [
+    'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 
+    'B', 'B',
+    'C', 'C',
+    'D', 'D', 'D', 'D', 
+    'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 
+    'F', 'F', 
+    'G', 'G', 'G', 
+    'H', 'H',  
+    'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 
+    'J',  
+    'K', 
+    'L', 'L', 'L', 'L', 
+    'M', 'M', 
+    'N', 'N', 'N', 'N', 'N', 'N', 
+    'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 
+    'P', 'P', 
+    'Q', 
+    'R', 'R', 'R', 'R', 'R', 'R', 
+    'S', 'S', 'S', 'S', 
+    'T', 'T', 'T', 'T', 'T', 'T',
+    'U', 'U', 'U', 'U', 
+    'V', 'V', 
+    'W', 'W', 
+    'X', 
+    'Y', 'Y', 
+    'Z'
+    ]
+    selected_index = []
+    picked_letters = []
+    for i in range(0, 10):
+        rand_selection = randint(0, len(tiles)-1)
+        while rand_selection in selected_index:
+            rand_selection = randint(0, len(tiles)-1)
+        selected_index.append(rand_selection)
+        random_letter = tiles[rand_selection]
+        picked_letters.append(random_letter)
+    return picked_letters
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    word_uppercase = word.upper()
+    letter_frequency_word = {}
+    letter_frequency_hand = {}
+    
+    for i in word_uppercase:
```

Comment:
Prefer to use a more descriptive variable than `i` here so that your code is more self-documenting. What does `i` represent here? 

Since the element we iterate over in this loop is a letter, something like `char` or `letter` could work.

```suggestion
    for letter in word_uppercase:
```